docs/jwst/ramp_fitting/description.rst

```diff
@@ -54,6 +54,11 @@ JWST-specific features are described later within this document.
 Upon successful completion of this step, the status keyword S_RAMP will be set
 to "COMPLETE".
 
+There is a new algorithm available for testing in ramp fitting, which is the
+likelihood algorithm.  It is selected by setting ``--ramp_fitting.algorithm=LIKELY``.
+The details are in the ``stcal`` documentation at
+:ref:`Likelihood Algorithm Details <stcal:likelihood_algo>`.
```

Comment:
This reference is throwing an error in the documentation build

Reply:
Shouldn't we expect this to fail, since the corresponding link doesn't exist yet?

Reply:
I think this should build once we merge the stcal PR and make that release.